packages/dd-trace/src/appsec/iast/overhead-controller.js

```diff
@@ -65,7 +65,8 @@ const SINGLE_SHOT_OPERATIONS = {
 }
 
 function hasQuotaSingleShot (operation, iastContext) {
-  if (!iastContext || !iastContext[OVERHEAD_CONTROLLER_CONTEXT_KEY]) {
+  const oceContext = iastContext && iastContext[OVERHEAD_CONTROLLER_CONTEXT_KEY]
+  if (!oceContext) {
     return false
   }
   return operation.hasQuota(iastContext[OVERHEAD_CONTROLLER_CONTEXT_KEY])
```

Comment:
You have create a var `oceContext` to use at this point and prevent the access to the object property